inc/class-blocks-animation.php

```diff
@@ -217,6 +222,202 @@ public static function add_frontend_anim_inline_style() {
+
+		/* translators: %s: Otter Blocks */
+		$notice_html .= '<button id="otter-animation-install-otter" class="button button-primary button-hero">';
+		$notice_html .= '<span class="dashicons dashicons-update hidden"></span>';
+		$notice_html .= '<span class="text">';
+		$notice_html .= 'installed' === $otter_status ?
+			/* translators: %s: Otter Blocks */
+			sprintf( __( 'Activate %s', 'otter-blocks' ), 'Otter Blocks' ) :
+			/* translators: %s: Otter Blocks */
+			sprintf( __( 'Install & Activate %s', 'otter-blocks' ), 'Otter Blocks' );
+		$notice_html .= '</span>';
+		$notice_html .= '</button>';
+
+		$notice_html .= '<a href="https://wordpress.org/plugins/otter-blocks/" target="_blank" class="button button-secondary button-hero">';
+		$notice_html .= '<span>' . __( 'Learn More', 'otter-blocks' ) . '</span>';
+		$notice_html .= '<span class="dashicons dashicons-external"></span>';
+		$notice_html .= '</a>';
+
+		$notice_html .= '</div>';
+
+		$notice_html .= '</div>';
+		$notice_html .= '</div>';
+		$notice_html .= '</div>';
+
+		echo wp_kses_post( $notice_html );
+	}
+
+	/**
+	 * Get the blocks animations plugin status.
+	 *
+	 * @return string
+	 */
+	private function get_blocks_animations_status() {
+		$status = 'not-installed';
+
+		if ( file_exists( ABSPATH . 'wp-content/plugins/otter-blocks/otter-blocks.php' ) ) {
+			return 'installed';
+		}
+
+		return $status;
+	}
+
+	/**
+	 * Should we show the welcome notice?
+	 *
+	 * @return bool
+	 */
+	private function should_show_welcome_notice() {
+		// Already using otter blocks.
+		if ( is_plugin_active( 'otter-blocks/otter-blocks.php' ) ) {
+			return false;
+		}
+
+		// Notice was dismissed.
+		if ( get_option( 'otter-animation-dismiss-welcome-notice', 'no' ) === 'yes' ) {
+			return false;
+		}
+
+		$screen = get_current_screen();
+
+		// Only show in dashboard/themes/plugins.
+		if ( ! in_array( $screen->id, array( 'dashboard', 'plugins', 'themes' ) ) ) {
+			return false;
+		}
+
+		// AJAX actions.
+		if ( defined( 'DOING_AJAX' ) && DOING_AJAX ) {
+			return false;
+		}
+
+		// Don't show in network admin.
+		if ( is_network_admin() ) {
+			return false;
+		}
+
+		// User can't dismiss. We don't show it.
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return false;
+		}
+
+		// User can't install plugins. We don't show it.
+		if ( ! current_user_can( 'install_plugins' ) ) {
+			return false;
+		}
+
+		// Block editor context.
+		if ( $screen->is_block_editor() ) {
+			return false;
+		}
+
+		// Dismiss after one week from activation.
+		$display_time = get_option( 'otter_animation_welcome_notice_display_time', false );
+		// Save notice display time to auto dismiss after a week.
+		if ( $display_time ) {
+			$display_time = time();
+			update_option( 'otter_animation_welcome_notice_display_time', $display_time );
+		}
+
+		if ( ! empty( $display_time ) && time() - intval( $display_time ) > WEEK_IN_SECONDS ) {
+			update_option( 'otter-animation-dismiss-welcome-notice', 'yes' );
```

Comment:
To keep with the rest of the plugin, we use snake_case for option names, and bool instead of yes/no.

Reply:
Done!